### ISSUE TITLE
An Introduction to Deep Generative Modeling

### ISSUE THREAD

**Issue by npresseller** [npresseller/Capstone-Bibliography#14]
### Title
An Introduction to Deep Generative Modeling
### URL
https://arxiv.org/abs/2103.05180
### Summary 
Discusses generative modeling as it related to deepfakes
### Key Points 
Deep generative modeling-
Deep generative models (DGM) are neural networks with many hidden layers trained to approximate
complicated, high-dimensional probability distributions using a large number of samples. When trained
successfully, we can use the DGMs to estimate the likelihood of each observation and to create new samples from the underlying distribution.

Variational Autoencoders-
In most practical situations, we cannot assume that the latent space dimension and that of the data
space are equal. This prohibits a direct use of the flow models from the previous section since the
generator is not invertible and the KL divergence may be unbounded or not well-defined [3]. Variational
Autoencoders (VAE) [29, 42, 30] are a popular framework to overcome this limitation and, typically, use
a latent space of much smaller dimension than the data space, that is, q  n. They also allow better
control of the latent space dimension as we see next. Since the generator, gθ, is not invertible, we cannot
compute the negative log-likelihood loss using (8) directly.

GANs-
     In generative adversarial networks (GAN), we train the weights of θ by minimizing a loss function that
measures the distance between gθ(Z) and X ; that is, GANs compare the distributions in the data space,
unlike CNFs and VAEs. Recall that X is represented by the training data and samples from gθ(Z)
obtained by transforming samples of the latent distribution. GANs are considered likelihood-free models
since they neither use the samples’ likelihood (as in the normalizing flows discussed in Section 3) nor
using a lower bound of the likelihood (as in the variational autoencoder presented in Section 4). Another
difference to the previous approaches is that GANs do not attempt to infer the latent variables that
underlie the samples. Many promising results [8, 26] have contributed to GANs’ increasing popularity,
and several excellent works that go beyond our short presentation are [19, 17, 2].

### Citation
@article{DBLP:journals/corr/abs-2103-05180,
  author    = {Lars Ruthotto and
               Eldad Haber},
  title     = {An Introduction to Deep Generative Modeling},
  journal   = {CoRR},
  volume    = {abs/2103.05180},
  year      = {2021},
  url       = {https://arxiv.org/abs/2103.05180},
  eprinttype = {arXiv},
  eprint    = {2103.05180},
  timestamp = {Mon, 15 Mar 2021 17:30:55 +0100},
  biburl    = {https://dblp.org/rec/journals/corr/abs-2103-05180.bib},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
